fp.js

```diff
@@ -23,7 +23,11 @@ people.forEach(person => {
     peopleYoungerThan35.push(person)
   }
 })
-
+var filtered = [
+  {name: "Jane", age: 32},
+  {name: "Jessica", age: 34},
+  {name: "Jasmine", age: 39}
+].filter(peopleYoungerThan35);
```

Comment:
In this example it looks like `peopleYoungerThan35` is still an empty array, so this won't do anything.

Filter is structured a lot like `map`. The main difference is that instead of returning what exactly you want to see in the new array, you are returning the condition that you want all items in the array to meet. So if we want to make sure that only people under 35 show up in the new array...

```js
people.filter(person => {
  return person.age < 35;
})
```